docs/hubs-controls.md

```diff
@@ -0,0 +1,662 @@
+---
+id: hubs-controls
```

Comment:
Are these docs generated or hand-written? How do we edit them if we change the underlying control scheme?

Reply:
hand-written (copied from from the existing wiki). I'd love to connect and see if there is a way to automatically sync these if they get changed.